### PR TITLE
Added a Remove button and its functionality.

### PR DESCRIPTION
**Pull request by Himanshunegi0332**
Targets #2213

### Description
 Added a remove functionality to the projects in the client.

### Checklist:
- [X] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Client/Resources/views/subviews/edit-client-projects.blade.php [hunk 1]
```diff
@@ -20,16 +23,23 @@
                             {{ $project->name }}
                         </a>
                     </div>
-                    <div class="w-33p w-md-40p ml-2">
+                    <div class="w-30p ml-2">
```
Comment (gautam071) [inline Modules/Client/Resources/views/subviews/edit-client-projects.blade.php:26]:
```suggestion
                    <div class="w-30p w-md-40p ml-2">
```

**Review by gautam071 — CHANGES_REQUESTED** [review]
Looks good one minor change

File: Modules/Client/Resources/views/subviews/edit-client-projects.blade.php (hunk 1 above)
Comment (Himanshunegi0332) [reply Modules/Client/Resources/views/subviews/edit-client-projects.blade.php]: done sir

**Comment by gautam071** [comment]
@Himanshunegi0332 Make the name and description more related to the PR

**Comment by gautam071** [comment]
@Himanshunegi0332 Sorry for changing the requirement at the last minute
Can we implement the soft delete

File: Modules/Client/Resources/views/subviews/edit-client-projects.blade.php [hunk 2]
```diff
@@ -20,22 +23,49 @@
...
+                                    </div>
+                                </div>
+                            </div> 
+                        </div>
+                    </div>
+                @endforeach
+            </div>
+            <div class="card-footer">
+                <a href="{{ route('client.index') }}" class="btn btn-primary btn-theme-gray-lighter mr-3">Exit</a>
+            </div>
```
Comment (gautam071) [inline Modules/Client/Resources/views/subviews/edit-client-projects.blade.php:68]: Can we move the modal to a new blade file and include the blade file here

File: Modules/Project/Http/Controllers/ProjectController.php [hunk 3]
```diff
@@ -78,6 +78,18 @@ public function show(Project $project)
         ]);
     }
 
+    public function destroy(ProjectRequest $request, Project $project)
+    {
+        Project::updateOrCreate(
+            [
+                'reason_for_deletion' => $request['comment']
```
Comment (gautam071) [inline Modules/Project/Http/Controllers/ProjectController.php:85]: Can you add backend validation not accepting any special characters

File: Modules/Client/Resources/views/subviews/edit-client-projects.blade.php [hunk 4]
```diff
@@ -20,22 +23,26 @@
...
         </div>
-    </div>
 
-</div>
\ No newline at end of file
+    </div>
```
Comment (gautam071) [inline Modules/Client/Resources/views/subviews/edit-client-projects.blade.php:48]: Leave a line at end of file

File: Modules/Client/Resources/views/subviews/create-client-projects-modal.blade.php [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+                        <label for="name"><span>Reason for Deletion</span></label>
+                        <input type="text" class="form-control" name="comment">
+                    </div>
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal">No</button>
+                    <button class="btn btn-primary" onclick="">Yes</button> 
+                </form>
+            </div>
+        </div>
+    </div>
+</div>
```
Comment (gautam071) [inline Modules/Client/Resources/views/subviews/create-client-projects-modal.blade.php:24]: Leave a line at end of file

**Review by gautam071 — APPROVED** [review]
Looks good! Left some suggestions

**Pull request merged**